update Search and Discover content with IN search (#3410)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/index.mdx

Also, the queries prioritize `AND` before `OR`. For example, "x `AND` y `OR` z" is the same as "(x `AND` y) `OR` z". Parentheses can be used to change the grouping. For example, "x `AND` (y `OR` z)".

### Multiple Values on the Same Key

> When you search multiple values on the same key without an operator, the search terms automatically use an `OR` operator. For example, "x:value1 x:value2" is the same as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.

Author: ```suggestion
When you search multiple values on the same key without an operator, the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" is the same as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.
```

> When you search multiple values on the same key without an operator, the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" is the same as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.

Reviewer: see previous edit

> When you search multiple values on the same key without an operator, the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" is the same as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.

Author: ```suggestion
When you search multiple values on the same key without an operator, the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" will find the the same results as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.
```

> When you search multiple values on the same key without an operator, the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" will find the the same results as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.

Author: ```suggestion
You can search multiple values for the same key by putting the values in a list. For example, "x:[value1, value2]" will find the the same results as "x:value1 `OR` x:value2". When you do this, the search returns issues/events that match any search term.
```

> An example of searching on the same key without an operator:

Author: ```suggestion
An example of searching on the same key with a list of values:
```

```

> release:12.0 release:13.0

Author: ```suggestion
release:[12.0, 13.0]
```

#### Exclusion

> By default, search terms with different keys use the `AND` operator; that is, they return the intersection of issues/events that match all search terms.

Author: ```suggestion
By default, search terms use the `AND` operator; that is, they return the intersection of issues/events that match all search terms.
```
